statsmodels/discrete/discrete_model.py

```diff
@@ -2543,14 +2543,16 @@ def _score_nbin(self, params, Q=0):
         y = self.endog[:,None]
         mu = self.predict(params)[:,None]
         a1 = 1/alpha * mu**Q
+        prob = a1 / (a1 + mu)  # a1 aka "size" in _ll_nbin
         if Q: # nb1
-            dparams = exog*mu/alpha*(np.log(1/(alpha + 1)) +
+            # Q == 1 --> a1 = mu / alpha --> prob = 1 / (alpha + 1)
```

Comment:
Right, but as I mentioned the last time I reviewed a PR for this code, `if Q` is satisfied by `Q = 2`.

Reply:
I can add an assertion that Q == 1 if that'll make you more comfortable.  I'd prefer not to because this is perf hotspot as it is, but so it goes.

Reply:
Why not `if Q == 1:`?